acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/NodeConfiguration.java

```diff
@@ -17,14 +17,16 @@
 public class NodeConfiguration {
 
   private static final String HTTP_URL_FORMAT = "http://%s:%s";
-  private static final int TCP_PORT = 8545;
-  private static final int WS_PORT = 8546;
+  private static final int TCP_PORT = 8547;
```

Comment:
why changed? Just to prevent port collision in some way?

Reply:
Meh, was seeing if the exposing actually worked (will be superseded by forthcoming work about next available port)